packages/common/src/Adaptor.js

```diff
@@ -932,3 +932,49 @@ export function assert(expression, errorMessage) {
     return state;
   };
 }
+
+/**
+ * Outputs a message to the console.
```

Comment:
Maybe:
```
Outputs a message, like calling `console.log`. Use this at the top level of your job code, but not inside callbacks.
```

And then actually maybe you'd add an example like:
```
// log a line inside a callback
fn((state) => {
   console.log(state.data)
   return state
})
```


All this because I'm worried that users will get confused over when to use `log` and when to use `console.log()`